test/lib/mayaUsd/utils/materialx_test_data/Channel1_topo_1_39.mtlx

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<materialx version="1.39">
+  <surfacematerial name="N0" type="material" nodedef="ND_surfacematerial">
+    <input name="surfaceshader" type="surfaceshader" nodename="N1" />
```

Comment:
Need new files since 1.39 does not have the concept of "channels", thus simplifying the test suite a lot.